src/pages/docs/octopus-cloud/frequently-asked-questions.md

```diff
@@ -195,21 +195,21 @@ Our Cloud Platform team observes uptime and planned downtime durations as part o
 
 We publish Octopus Cloud's [operational status](https://status.octopus.com/). You can subscribe to status change emails from our [operational status page](https://status.octopus.com/).
 
-## Logs & Data
+## Logs and data
 
 ### How do we access deployment and audit logs? 
 
-Deployment and access logs can be viewed and downloaded from Octopus Cloud. Enterprise-tier customers can [stream audit logs](/docs/security/users-and-teams/auditing/audit-stream) to their SIEM system.
+You can view and download deployment and access logs from Octopus Cloud. Enterprise-tier customers can [stream audit logs](/docs/security/users-and-teams/auditing/audit-stream) to their SIEM system.
```

Comment:
@adrianparker I wonder if it would be helpful to specify where exactly people can view and download deployment and access logs from Octopus Cloud, or to link somewhere that explains that?

Reply:
I'll make that improvement shortly, thank you, great suggestion.